fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -123,6 +115,33 @@ def self.start(user: nil, is_swift_fastfile: false)
     end
     # rubocop:enable Metrics/BlockNesting
 
+    def self.already_set_up(user: nil, is_swift_fastfile: false)
+      if is_swift_fastfile && !FastlaneCore::FastlaneFolder.swift?
+        UI.important("fastlane is already configured using Ruby.")
+        UI.message("If you want to switch to Swift, I can reinitialize your fastlane project:")
+        UI.message(" 1️⃣  The fastlane directory (at #{FastlaneCore::FastlaneFolder.path}) would be renamed `fastlane.old`")
+        UI.message(" 2️⃣  I would run the fastlane Swift initialization")
+        UI.message(" 3️⃣  You'd need to port your customizations from Ruby to Swift by hand.")
```

Comment:
what "customizations" does this refer to? my own lanes in the Fastfile?

Reply:
Yeah, anything in a Fastfile, Matchfile, Appfile, etc. which would typically be parsed as Ruby.

Reply:
Can we somehow formulate this in a way that makes it clearer? I though of customizations in fastlane first - which of course no one really does. Or is this just me?